### DISCUSSION TITLE
HDR and float support

### DISCUSSION THREAD

**Discussion by Raildex** [SixLabors/ImageSharp#2591] [Ideas]
ImageSharp currently supports TIFF, which actually supports more than 8 bits per channel and also supports floating point values. But ImageSharp only supports encoding to 8 bits. THis is fairly lossy when you need to save images on disk for temporary storage.

I would also like to have HDR support with the file HDR and EXR file formats

**Comment by Raildex** [comment]
Any news on this?

**Reply by JimBobSquarePants** [reply]
Not at the moment. Feel free to submit a PR.
